Remove install upgrade ph (#153)
mulesoft/docs-private-cloud: modules/ROOT/pages/upgrade.adoc

= Upgrade Workflow
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

> == MuleSoft Professional Services Required

Reviewer: Should the following text mimic that used in the previous topic with the same title?

Reviewer: ditto MuleSoft Professional Services, GLOBALLY. I will stop marking.

Contact MuleSoft professional services prior to upgrading. MuleSoft professional services must be involved throughout the upgrade process. 

> Before an Anypoint Platform Private Cloud Edition (Anypoint Platform PCE) upgrade is performed, you must verify that your working installation meets any additional requirements of the new Anypoint Platform PCE version, such as opening additional ports or adding a new port for the load balancer.

Reviewer: ```suggestion
Before you upgrade Anypoint Platform Private Cloud Edition (Anypoint Platform PCE), verify that your working installation meets any additional requirements of the most recent Anypoint Platform PCE version, such as opening additional ports or adding a new port for the load balancer.
```

include::_attributes.adoc[]
endif::[]

== Before You Begin

> Because Anypoint Platform Private Cloud Edition (Anypoint Platform PCE) upgrades require a working installation, all environment prerequisites should already be met. As long as you (or your infrastructure team) successfully performed all prerequisite tasks listed in xref:prereq-hardware.adoc[Hardware Prerequisites], xref:prereq-software.adoc[Software Prerequisites], and xref:prereq-network.adoc[Network Prerequisites] prior to the installation of Anypoint Platform PCE, your environment can be upgraded successfully.

Reviewer 2: Before the upgrade is ran, the requirements need to be verified against the requirements for the new version as sometimes there are small changes (usually opening new ports, sometimes adding a new port in the load balancer).
The recommendation is to re-verify that the environment is ready against the requirements for the new version.

Author: Added new wording.

> Contact MuleSoft professional services prior to uninstalling. MuleSoft professional services must be involved throughout the upgrade process. 

Reviewer 2: prior to ~uninstalling.~ -> `upgrading.`

Because Anypoint Platform Private Cloud Edition (Anypoint Platform PCE) upgrades require a working installation, all environment prerequisites should already be met. As long as you (or your infrastructure team) successfully performed all prerequisite tasks listed in xref:prereq-hardware.adoc[Hardware Prerequisites], xref:prereq-software.adoc[Software Prerequisites], and xref:prereq-network.adoc[Network Prerequisites] prior to the installation of Anypoint Platform PCE, your environment can be upgraded successfully.

> Refer to the xref:release-notes::pce/anypoint-private-cloud-release-notes.adoc[Anypoint Platform Private Cloud Edition Release Notes] for additional information about upgrading Anypoint Platform PCE to the most recent version.

Reviewer 3: I think in the release notes we removed the additional information about upgrading PCE. What about changing this line to:

Refer to the xref:release-notes::pce/anypoint-private-cloud-release-notes.adoc[Anypoint Platform Private Cloud Edition Release Notes] for additional information about the most recent Anypoint Platform PCE version.

Author: Makes sense - just updated.

endif::[]

> == Before You Begin

Reviewer 3: Maybe remove this header?

Author: I added a statement that resembles the install one - please take a look.

> [IMPORTANT]

Reviewer 3: Similar to the install page, we could potentially make this its own section for emphasis.

### MuleSoft Professional Services Required
MuleSoft professional services must be involved throughout the installation process. 

Because Anypoint Platform Private Cloud Edition (Anypoint Platform PCE) upgrades require a working installation, all environment prerequisites should already be met. As long as you (or your infrastructure team) successfully performed all prerequisite tasks listed in xref:prereq-hardware.adoc[Hardware Prerequisites], xref:prereq-software.adoc[Software Prerequisites], and xref:prereq-network.adoc[Network Prerequisites] prior to the installation of Anypoint Platform PCE, your environment can be upgraded successfully.

Author: Got it - thanks!
